generate-dist-links.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+TARGET=build/tmp/python/hail/docs/distLinks.rst
+rm -f $TARGET
+
+HAIL_VERSION=$1
+HASH=$(git rev-parse --short HEAD)
+
+
+echo "Hail uploads distributions to Google Storage as part of our continuous integration suite." >> $TARGET
+
+echo "You can download a pre-built distribution from the below links. Make sure you download the distribution that matches your Spark version!" >> $TARGET
```

Comment:
looks like you might be short one newline compared to the current text. Is that correct?